### PR TITLE
Support decimals

### PR DESCRIPTION
**Pull request by claresloggett**
Allow user to use decimals on command line

### CODE REVIEW

File: calc.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import sys
 
 filename, num1, op, num2 = sys.argv
-num1, num2 = int(num1), int(num2)
+num1, num2 = float(num1), int(num2)
```
Comment (jni) [inline calc.py]: This is a nice change! Can you also add support for the second int?

**Review by jni — CHANGES_REQUESTED** [review]
Thanks @claresloggett! I've left a minor comment but otherwise awesome!

File: calc.py (hunk 1 above)
Comment (claresloggett) [reply calc.py]: Sure!

**Review by jni — APPROVED** [review]

**Comment by jni** [comment]
Awesome thank you!!

**Pull request merged**